scripts/gen-current-build.sh

```diff
@@ -9,7 +9,7 @@ if [[ $? -ne 0 ]]; then
 	exit 1
 fi
 
-COMPILER_INFO="$($CPLUSPLUS -v 2>&1)"
+COMPILER_INFO="$($CPLUSPLUS --version 2>&1)"
```

Comment:
Good point. We can move back to `-v` upon cleaning up `build_info` from the stream/log of Karl.